rest-service/manager_rest/rest/resources_v3_1/deployments.py

```diff
@@ -236,17 +236,20 @@ def put(self, deployment_id, **kwargs):
         with sm.transaction():
             if not skip_plugins_validation:
                 rm.check_blueprint_plugins_installed(blueprint.plan)
-            deployment = rm.create_deployment(
-                blueprint,
-                deployment_id,
-                private_resource=args.private_resource,
-                visibility=visibility,
-                site=site,
-                runtime_only_evaluation=request_dict.get(
+            dep_args = {
+                "private_resource": args.private_resource,
+                "visibility": visibility,
+                "site": site,
+                "runtime_only_evaluation": request_dict.get(
                     'runtime_only_evaluation', False),
-                created_at=created_at,
-                created_by=owner,
-            )
+                "created_at": created_at,
+                "created_by": owner,
+            }
+            workflows = request_dict.get('workflows')
+            if workflows:
+                dep_args['workflows'] = workflows
+            deployment = rm.create_deployment(blueprint, deployment_id,
+                                              **dep_args)
```

Comment:
you don't need to rewrite all this, `rm.create_deployment` has a `if workflows`, so just a `rm.create_deployment(..., workflows=request_dict.get('workflows')` here works

Reply:
Good point, will update.